libs/docs/cdk/focusable-grid/focusable-grid-header/focusable-grid-header.component.html

```diff
@@ -0,0 +1,11 @@
+<header>Focusable Grid</header>
+<description>
+    You can quickly build out focusable grid collection for your components. For that you will need to use a combination
+    of <code>[fdkFocusableGrid]</code>, <code>[fdkFocusableList]</code> and <code>[fdkFocusableItem]</code> directives.
+    It also respects <code>[fdkDisabled]</code> and <code>[fdkReadonly]</code> directives, so that those elements will
```

Comment:
Typical behavior for readonly elements (such as inputs) still allows them to be focused and reached via tab keys - so maybe they should not be excluded from the arrow keys as well?

Reply:
Yea, I agree with you, let me change it here and in focusable list.